angular-spa-ui/js/search-app/search-module/index.jade

```diff
@@ -0,0 +1 @@
+include search.html
```

Comment:
Did you review the pull request? Or this red arrow and sign are visible only for me?